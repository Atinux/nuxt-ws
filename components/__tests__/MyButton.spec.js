import { shallowMount } from '@vue/test-utils'
import MyButton from '../MyButton'

const factory = () => shallowMount(MyButton, {})

describe('MyButton', () => {
  test('mounts properly', () => {
    const wrapper = factory()
    expect(wrapper.isVueInstance()).toBeTruthy()
  })

  test('renders properly', () => {
    const wrapper = factory()
    expect(wrapper.html()).toMatchSnapshot()
  })
})
