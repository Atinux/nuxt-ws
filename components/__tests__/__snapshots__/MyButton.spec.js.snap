// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MyButton renders properly 1`] = `<button>Button</button>`;
